tests/integration.rs

```diff
@@ -165,27 +168,31 @@ pub fn ed25519() {
 #[test]
 #[cfg(feature = "alloc")]
 fn critical_extensions() {
-    let root = include_bytes!("critical_extensions/root-cert.der");
-    let ca = include_bytes!("critical_extensions/ca-cert.der");
+    let root = CertificateDer::from(&include_bytes!("critical_extensions/root-cert.der")[..]);
+    let ca = CertificateDer::from(&include_bytes!("critical_extensions/ca-cert.der")[..]);
 
     let time = webpki::Time::from_seconds_since_unix_epoch(1_670_779_098);
-    let anchors = [webpki::TrustAnchor::try_from_cert_der(root).unwrap()];
+    let anchors = [extract_trust_anchor(&root).unwrap()];
 
-    let ee = include_bytes!("critical_extensions/ee-cert-noncrit-unknown-ext.der");
-    let res = webpki::EndEntityCert::try_from(&ee[..]).and_then(|cert| {
+    let ee = CertificateDer::from(
+        &include_bytes!("critical_extensions/ee-cert-noncrit-unknown-ext.der")[..],
+    );
+    let res = webpki::EndEntityCert::try_from(&ee).and_then(|cert| {
         cert.verify_for_usage(
             ALL_SIGALGS,
             &anchors,
-            &[ca],
+            &[ca.clone()],
```

Comment:
this clone is ~free, right? because `ca` will be a `CertificateDer(Der::Borrowed(_))`?

Reply:
Yup, in this case `ca` should be a `CertificateDer<'static>` holding a `&'static [u8]`.